web/gui-v2/src/static_data/table_columns.js

```diff
@@ -81,23 +87,50 @@ const columnDefinitions = [
   {
     title: "Region",
     key: "continent",
-    initialCol: true,
+    initialCol: false,
     minWidth: 170,
     type: 'dropdown',
   },
   {
     title: "Stage",
     key: "stage",
-    initialCol: true,
+    initialCol: false,
     minWidth: 120,
     type: 'dropdown',
   },
+  // TODO, pending #120 adding `sector` data
+  // {
+  //   title: "Sector",
+  //   key: "sector",
+  //   initialCol: false,
+  //   minWidth: 200,
+  //   type: 'dropdown',
+  // },
 
   {
     title: "All publications",
     key: "all_pubs",
     ...generateSliderColDef("articles", "all_publications"),
   },
+  {
+    title: "5-year growth in publications",
+    key: "all_pubs_growth",
+    ...generateSliderColDef(
+      "articles",
+      "all_publications",
+      ((_val, row) => {
+        const data = row.articles.all_publications;
+        const startVal = data.counts[startArticleIx];
+        return Math.round((data.counts[endArticleIx] - startVal) / startVal * 1000) / 10;
```

Comment:
Need to handle 0 startVal

<img width="1792" alt="Screenshot 2023-10-31 at 11 17 27 AM" src="https://github.com/georgetown-cset/parat/assets/1252409/4e8d958f-7f90-4b7c-a25b-b7f89ce057eb">

Reply:
That's one of the questions from https://github.com/georgetown-cset/parat/issues/92#issuecomment-1765118282:

> > * 5-year growth in papers
>
> I can calculate the growth rate, but then we wouldn't have rankings. If we want rankings in this column (and the similar patents column) then this stat should be pre-calculated and included in the data
>
> Related question: how should infinite growth be handled (i.e. 0 papers in the start year, and some number >0 in the end year)? Right now these rows say "Infinity%".

Reply:
I'll pre-calculate this for you and include the rank